scripts/obfi/hide.py

```diff
@@ -28,8 +29,9 @@ def __init__(self) -> None:
     def get_seed(self) -> None:
         """Get the day's seed."""
         try:
-            with urllib.request.urlopen(SEED_PATH) as handle:
-                content = handle.read()
+            r = requests.get(SEED_PATH)
+            r.raise_for_status()
+            content = r.text
```

Comment:
This needs to be bytes
```suggestion
            content = r.content
```